assets/src/design-system/theme/helpers/outline.js

```diff
@@ -23,11 +23,12 @@ import { css } from 'styled-components';
  */
 import { FOCUS_VISIBLE_SELECTOR } from '../global';
 
-export const focusableOutlineCSS = (accent) => css`
-  border: solid 2px transparent;
-
-  &.${FOCUS_VISIBLE_SELECTOR} {
-    outline: none;
-    border: solid 2px ${accent};
-  }
-`;
+export const focusableOutlineCSS = (v) => {
+  const accent = typeof v === 'string' ? v : v?.theme?.colors?.border?.focus;
+  return css`
+    &.${FOCUS_VISIBLE_SELECTOR} {
+      outline: none;
+      box-shadow: 0px 0px 0px 2px ${accent};
```

Comment:
@BrittanyIRL is this fine? 

I remember we moved away from box-shadow when we needed the gap between the focus style and the element focused, but then we removed the design need to have that gap. Anyway, I think this is less intrusive on element styles, retains the corner radius and doesn't cause the content shift like I was getting before with outline.

Let me know if I'm missing a necessity for using outline & I'd be happy to revert back

Reply:
So, we actually want to use the border because this is in play for conditionally colored outlines too - like errors - or manipulating perceived focus like on the dropDown. 

I suppose we could update the usage of it so that we're passing in an updated accent color to use instead of swapping the border-color conditionally in the styles.

Reply:
No no. this is exactly the necessity I was asking for. Good catch! thanks

Not worth disrupting any systems already built off of this imo. Will revert back